nncf/torch/quantization/strip.py

```diff
@@ -163,14 +178,159 @@ def remove_disabled_quantizers(model: NNCFNetwork) -> NNCFNetwork:
     return model
 
 
-def strip_quantized_model(model: NNCFNetwork):
+def strip_quantized_model(model: NNCFNetwork, strip_format: StripFormat = StripFormat.NATIVE):
     """
-    Returns the model with as much custom NNCF additions as possible removed
-    while still preserving the functioning of the model object as a compressed model.
+    Removes auxiliary layers and operations added during the quantization process,
+    resulting in a clean quantized model ready for deployment. The functionality of the model object is still preserved
+    as a compressed model.
 
     :param model: Compressed model.
+    :param strip format: Describes the format in which model is saved after strip.
     :return: The modified NNCF network.
     """
-    model = replace_quantizer_to_torch_native_module(model)
-    model = remove_disabled_quantizers(model)
+    if strip_format == StripFormat.DQ:
+        model = replace_with_decompressors(model)
+    elif strip_format == StripFormat.NATIVE:
+        model = replace_quantizer_to_torch_native_module(model)
+        model = remove_disabled_quantizers(model)
+    else:
+        msg = f"Unsupported strip format: {strip_format}"
+        raise nncf.ParameterNotSupportedError(msg)
     return model
+
+
+def replace_with_decompressors(model: NNCFNetwork) -> NNCFNetwork:
+    """
+    Performs transformation from fake quantize format (FQ) to dequantization one (DQ).
+    The former takes floating-point input, quantizes and dequantizes, and returns a floating-point value,
+    while the latter takes a quantized integer representation, dequantizes it, and outputs a floating-point result.
+
+    Mathematically, both methods lead to the same outcome, but due to differences in the order of operations and
+    rounding errors, the actual results may differ. In particular, this error can occur for values
+    that are located in the midpoint between two quantized values ("quants").
+
+    The FQ format may round these values to one "quant", while the DQ format rounds them to another "quant".
+    To avoid these issues, the compressed representation should be provided not by directly quantizing the input,
+    but by quantizing a pre-processed, fake-quantized, floating-point representation.
+
+    :param model: Compressed model with Decompressors.
+    :return: The modified NNCF network.
+    """
+    transformation_layout = TransformationLayout()
+    transformations = model.nncf.transformation_layout().transformations
```

Comment:
As fast solution is it may be ok, but using transformation is WA to replacer module. 
It's not portable to new tracing, potentially problems other problems in changes in other parts of code that not related to strip.

As i see need to do two steps: replace quantizer and pack wights.

As fast WA to fast merge works is it ok, but it should be reworked after.

Reply:
Strip will have slightly different implementation with new tracing. Decided to support with old tracing first.